Resources/config/provider-orm.xml

```diff
@@ -12,7 +12,7 @@
 
     <services>
 
-        <service id="cmf_routing.content_repository" class="%cmf_routing.orm.content_repository.class%">
+        <service id="cmf_routing.orm_content_repository" class="%cmf_routing.orm.content_repository.class%">
```

Comment:
I followed the same name pattern than in provider-phpcr.xml (https://github.com/symfony-cmf/RoutingBundle/blob/master/Resources/config/provider-phpcr.xml#L33)